src/sql_generator.cpp

```diff
@@ -495,11 +505,17 @@ void truncate_table(duckdb::Connection &con, const table_def &table,
     sql << "UPDATE " << absolute_table_name << " SET "
         << KeywordHelper::WriteQuoted(deleted_column, '"') << " = true";
   }
+  mdlog::info("truncate_table request: synced column = " + synced_column);
   sql << " WHERE " << KeywordHelper::WriteQuoted(synced_column, '"')
       << " < make_timestamp(?)";
   auto query = sql.str();
+  const std::string err =
+      "Error truncating table at bind step <" + absolute_table_name + ">";
   mdlog::info("truncate_table: " + query);
   auto statement = con.Prepare(query);
+  if (statement->HasError()) {
+    throw std::runtime_error(err + " (at bind step):" + statement->GetError());
```

Comment:
Here we say `at bind step` twice, effectively:
`"Error truncating table at bind step <" + absolute_table_name + ">" + `" (at bind step):"`